libr/anal/cc.c

```diff
@@ -104,6 +113,20 @@ R_API const char *r_anal_cc_arg(RAnal *anal, const char *convention, int n) {
 	return ret? r_str_constpool_get (&anal->constpool, ret): NULL;
 }
 
+R_API const char *r_anal_cc_self(RAnal *anal, const char *convention) {
+	r_return_val_if_fail (anal && convention, NULL);
+	char *query = sdb_fmt ("cc.%s.self", convention);
```

Comment:
What about calling it "this" instead of "self"? I think it's slightly more common, especially considering the term "thiscall".